### PR TITLE
fix: remove unused proxyPort option

### PR DESCRIPTION
**Pull request by olav**

### CODE REVIEW

**Review by chriswk — APPROVED** [review]
LGTM

File: .eslintrc [hunk 1]
```diff
@@ -46,7 +46,7 @@
           "files": ["*.ts", "*.tsx"],
           "rules": {
               "@typescript-eslint/explicit-module-boundary-types": ["error"],
-              "@typescript-eslint/indent": ["error"],
```
Comment (sighphyre) [inline .eslintrc:49]: Intentional?

File: README.md [hunk 2]
```diff
@@ -103,40 +103,40 @@ Keep-Alive: timeout=5
 
 ### Available options
 
-| Option          | Environment Variable      | Default value | Required |  Description  |
-| -------------   |----------------------     |----------     |:--------:|---------------|
-| unleashUrl      |`UNLEASH_URL`              | n/a           | yes      | API Url to the Unleash instance to connect to |
-| unleashApiToken | `UNLEASH_API_TOKEN`       | n/a           | yes      | API token (client) needed to connect to Unleash API. |
-| clientKeys      | `UNLEASH_PROXY_CLIENT_KEYS`     | n/a           | yes      | List of client keys that the proxy should accept. When querying the proxy, Proxy SDKs must set the request's _client keys header_ to one of these values. The default client keys header is `Authorization`. |
-| proxySecrets    | `UNLEASH_PROXY_SECRETS`   | n/a           | no      | Deprecated alias for `clientKeys`. Please use `clientKeys` instead. |
-| proxyPort       | `PORT`                    | 3000          | no       | The port where the proxy should listen. |
```
Comment (sighphyre) [inline README.md:112]: So I can't add a suggestion on a deleted line but what about using `n/a` instead of just removing it?

**Review by sighphyre — APPROVED** [review]
Some small questions but looks good!

File: .eslintrc (hunk 1 above)
Comment (olav) [reply .eslintrc:49]: Yeah, seems like this rule conflicts with prettier.

Comment (sighphyre) [reply .eslintrc:49]: Make sense!

**Pull request merged**